docs/asciidoc/modules/ROOT/pages/graph-updates/uuid.adoc

```diff
@@ -100,46 +109,80 @@ Add the `uuid`:
+| label    | installed | properties
+| "Person" | true      | {uuidProperty -> "uuid", addToExistingNodes -> true}
 |===
 
-The result is Node Person that has 2 properties:
+It is then possible to execute the following query, after a time defined by the configuration `apoc.uuid.refresh`:
+[source,cypher]
+----
+CREATE (n:Person {name: 'Daniel'})-[:Work]->(:Company {name: 'Neo4j'})
+----
+
+and the result will be a node `:Person` with 2 properties:
 
 image::apoc.uuid.result.png[width=800]
 
-Get all the uuid installed, call the procedure as:
+If the default configuration `addToExistingNodes: true` is chosen,
+in background (via the `apoc.periodic.iterate` procedure) all existing nodes will also be populated with a property with uuid values.
+and, when the execution is finished,
+a log with the query result like the following will be printed:
+```
+Result of batch computation obtained from existing nodes for UUID handler with label `MyLabel`:
+{failedParams={}, committedOperations=1, batch={total=10, committed=10, failed=0, errors={}}, wasTerminated=false, batches=1, timeTaken=0, retries=0, errorMessages={}, total=1, operations={total=10, committed=10, failed=0, errors={}}, failedOperations=0, updateStatistics={nodesDeleted=0, labelsAdded=0, relationshipsCreated=0, nodesCreated=0, propertiesSet=1, relationshipsDeleted=0, labelsRemoved=0}, failedBatches=0}
+```
+
 
+==== List of Automatic UUIDs
+
+It is possible to return the full list of automatic UUIDS in a database. For example, if the UUID in the following query is created:
 [source,cypher]
 ----
-CALL apoc.uuid.list()
-YIELD label, installed, properties
-RETURN label, installed, properties
+CALL apoc.uuid.create('neo4j', 'TestShow')
 ----
 
-The result is:
+It is then possible to run (also in this case, after a time defined by the configuration `apoc.uuid.refresh`):
 
-[opts="header",cols="1,1,1"]
+[source,cypher]
+----
+CALL apoc.uuid.show('neo4j')
+----
+
+.Results
+
+[opts="header",cols="1,1,1,1"]
 |===
 | label    | installed | properties
-| "Person" | true      | {uuidProperty -> "uuid", addToExistingNodes -> true}
+| "Person" | true      | {uuidProperty -> "uuid", addToExistingNodes -> true} | {wasTerminated -> false, count -> 10, batches -> 1, successes -> 1, failedOps -> 0, timeTaken -> 0, operationErrors -> {}, failedBatches -> 0}
 |===
 
 
-Remove the uuid installed call the procedure as:
+[WARNING]
+====
+Please note that, since the auto-UUID operations are eventually consistent (based on the `apoc.uuid.refresh` configuration),
+the `apoc.uuid.show` may return some UUIDs not yet added/updated/removed.
+To get the list of all of currently installed UUIDs,
+use the xref::overview/apoc.uuid/apoc.uuid.list.adoc[apoc.uuid.list] against the session database ("neo4j" in the above case).
+====
+
+
+==== Remove Automatic UUIDs
 
 [source,cypher]
 ----
-CALL apoc.uuid.remove('Person')
```

Comment:
why didn't we just add the database as the last parameter to remove and default it to neo4j?